common/test/acceptance/tests/lms/test_lms_cohorted_courseware_search.py

```diff
@@ -237,7 +244,7 @@ def test_cohorted_search_user_c_ab_content(self):
         """
         Test user not enrolled in any cohorts can't see any of restricted content.
         """
-        self._auto_auth(self.USERNAME, self.EMAIL, False)
+        self._auto_auth(self.cohort_default_student_username, self.cohort_default_student_email, False)
```

Comment:
I would change the method name from to
`test_cohorted_search_user_default_ab_content` for consistency.